### ISSUE TITLE
Logging module setting wrong output level

### ISSUE THREAD

**Issue by schemen** [schemen/AtmoPi#1]
The set level in config.ini is not influencing the output level. It seems to be in debugging mode all the time.

**Comment by wesselsd** [comment]
This is caused by the initialize_logger function in utils.py:

```python
    if config["DEBUG"]:
        outputlevel = "debug"
        logger.setLevel(logging.DEBUG)
```
this branch will always be taken since config["DEBUG"] evaluates to the string 'False' or the string 'True' - those both have the boolean value True.

**Comment by schemen** [comment]
Fixed with #4
